harness/determined/keras/_tensorboard_callback.py

```diff
@@ -6,6 +6,13 @@
 
 
 class TFKerasTensorBoard(tf.keras.callbacks.TensorBoard):  # type: ignore
+    """
+    This is a thin wrapper over the TensorBoard callback that ships with ``tf.keras``.  For more
```

Comment:
non-blocking: Might be worth mentioning that the `log_dir` parameter is ignored.